iungo-web/img-compressor-cli/0.1/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+	"copy-from-package": {
+		"bin/compressor": ""
```

Comment:
Shouldn't this be handled by composer? See https://getcomposer.org/doc/articles/vendor-binaries.md

Reply:
I also thought about doing it this way, but the point is that this system is something to use alone through the command line, so I thought it was more appropriate to put the executable file at the root of the project.

Reply:
But if the user explicitly says that they want all the binaries installed by composer in a specific binary directory, I think that should be respected.